components/bubble/index.en-US.md

```
@@ -39,23 +40,41 @@ Common props ref：[Common props](/docs/react/common-props)
-| footer | Footer content | React.ReactNode \| (content: [BubbleContentType](https://github.com/ant-design/x/blob/d3232c925a0dc61ad763c6664e16f07323ebca4a/components/bubble/interface.ts#L21)) => React.ReactNode | - |  |
-| header | Header content | React.ReactNode | - |  |
+| avatar | Avatar component | React.ReactNode | - | - |
+| classNames | Semantic DOM class | [Record<SemanticDOM, string>](#semantic-dom) | - | - |
+| content | Content of bubble | ContentType | - | - |
+| footer | Footer content | React.ReactNode \| (content: ContentType, info:{ key?: string \| number }) => React.ReactNode | - | - |
+| header | Header content | React.ReactNode \| (content: ContentType, info:{ key?: string \| number }) => React.ReactNode | - | - |
 | loading | Loading state of Message | boolean | - |  |
 | placement | Direction of Message | `start` \| `end` | `start` |  |
-| shape | Shape of bubble | `round` \| `corner` | - |  |
-| styles | Semantic DOM style | [Record<SemanticDOM, CSSProperties>](#semantic-dom) | - |  |
+| shape | Shape of bubble | `round` \| `corner` | - |  |  | styles | Semantic DOM style | [Record<SemanticDOM, CSSProperties>](#semantic-dom) | - |  |
```

markdownlint-cli2 (0.17.2)

Table column count
Expected: 5; Actual: 11; Too many cells, extra data will be missing

(MD056, table-column-count)